### PR TITLE
Storybook integration using webpack

### PR DESCRIPTION
**Pull request by yoching**
Added storybook integration using webpack!

If this approach looks good, I'll add more samples.

RelatedPR: https://github.com/unisonweb/ui-core/pull/55

## Current samples

Screenshot 2023-03-13 at 16 35 24
Screenshot 2023-03-13 at 16 35 27
Screenshot 2023-03-13 at 16 35 30

### CODE REVIEW

File: storybook/.npmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+legacy-peer-deps=true
```
Comment (hojberg) [inline storybook/.npmrc:1]: Whats this needed for?

File: storybook/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    "@storybook/builder-webpack5": "^6.5.16",
+    "@storybook/html": "^6.5.16",
+    "@storybook/manager-webpack5": "^6.5.16",
+    "@storybook/testing-library": "^0.0.13",
+    "babel-loader": "^8.3.0",
+    "elm-webpack-loader": "^8.0.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
+  }
+}
```
Comment (hojberg) [inline storybook/package.json:29]: Can we remove the fields we don't need:
`version`, `description`, `main`, `scripts.test`, `keywords`, `author`.

Also change license to `MIT` please.

File: storybook/stories/Helpers/style.css [hunk 3]
```diff
@@ -0,0 +1 @@
+.col { display: flex; flex-direction: column; }
```
Comment (hojberg) [inline storybook/stories/Helpers/style.css:1]: lets do this in the same style as the other css, TLDR; run with prettier.

**Review by hojberg — CHANGES_REQUESTED** [review]
Awesome. A few minor notes and can we also run prettier on the JS files please.

**Comment by hojberg** [comment]
I've updated the `main` branch with a prettier setup.

File: storybook/.npmrc (hunk 1 above)
Comment (yoching) [reply storybook/.npmrc]: This file appeared when I ran webpack & npm migration command from storybook.
But it seems to work without this setting. So I deleted it.

File: storybook/stories/Helpers/style.css (hunk 3 above)
Comment (yoching) [reply storybook/stories/Helpers/style.css]: I ran prettier and pushed🙂

**Comment by yoching** [comment]
Please see this PR for CI errors.
https://github.com/unisonweb/ui-core/pull/59

File: storybook/stories/Stories/Code/WorkspaceItem.elm [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+
+
+view : Model -> Html Msg
+view model =
+    let
+        decodeResult : Result Json.Decode.Error (List Item)
+        decodeResult =
+            let
+                result =
+                    decodeString (decodeList sampleReference) incrementGetDefinitionResponse
```
Comment (yoching) [inline storybook/stories/Stories/Code/WorkspaceItem.elm:47]: @hojberg 
Do you think this way, which is to decode hard-coded JSON strings, is ok for now?
I couldn't find a way to make entities manually, without changing the access controls (`exposing`s) in ui-core. Json string decoding was the only way I could find to make entities.

When we want to see other codes, we can replace the JSON string with others, which we can get by accessing UCM API.

Comment (hojberg) [reply storybook/stories/Stories/Code/WorkspaceItem.elm:47]: Yeah it seems ok for now

**Review by hojberg — APPROVED** [review]

**Comment by hojberg** [comment]
Merged your PR for the CI fix, after a rebase this should be good to go 🎉

**Comment by hojberg** [comment]
@yoching Is this ok to mark as ready?

**Comment by yoching** [comment]
Ok! I was going to add a bit more samples to this, but I can do it later with another PR.

**Comment by hojberg** [comment]
Yay!! 🎉🎉

**Pull request merged**